content/docs/replication/_index.md

```diff
@@ -18,40 +18,40 @@ CSM for Replication provides the following capabilities:
 {{</table>}}
 
 
 ## Supported Operating Systems/Container Orchestrator Platforms
 
 {{<table "table table-striped table-bordered table-sm">}}
-| COP/OS            | PowerMax         | PowerStore       | PowerScale       |
-| ----------------- | ---------------- | ---------------- | ---------------- |
-| Kubernetes        | 1.24, 1.25, 1.26 | 1.24, 1.25, 1.26 | 1.24, 1.25, 1.26 |
-| Red Hat OpenShift | 4.10, 4.11       | 4.9, 4.10        | 4.9, 4.10        |
-| RHEL              | 7.x, 8.x         | 7.x, 8.x         | 7.x, 8.x         |
-| CentOS            | 7.8, 7.9         | 7.8, 7.9         | 7.8, 7.9         |
-| Ubuntu            | 20.04            | 20.04            | 20.04            |
-| SLES              | 15SP4            | 15SP2            | 15SP2            |
+| COP/OS            | PowerMax         | PowerStore       | PowerScale       | PowerFlex        |
+| ----------------- | ---------------- | ---------------- | ---------------- | ---------------- |
+| Kubernetes        | 1.24, 1.25, 1.26 | 1.24, 1.25, 1.26 | 1.24, 1.25, 1.26 | 1.24, 1.25, 1.26 |
+| Red Hat OpenShift | 4.10, 4.11       | 4.9, 4.10        | 4.9, 4.10        | 4.9, 4.10        |
+| RHEL              | 7.x, 8.x         | 7.x, 8.x         | 7.x, 8.x         | 7.x, 8.x         |
+| CentOS            | 7.8, 7.9         | 7.8, 7.9         | 7.8, 7.9         | 7.8, 7.9         |
+| Ubuntu            | 20.04            | 20.04            | 20.04            | 20.04            |
+| SLES              | 15SP4            | 15SP2            | 15SP2            | 15SP3            |
 {{</table>}}
 
 ## Supported Storage Platforms
 
 {{<table "table table-striped table-bordered table-sm">}}
-|               |                                                 PowerMax                                                 |     PowerStore      |             PowerScale             |
-| ------------- | :------------------------------------------------------------------------------------------------------: | :-----------------: | :--------------------------------: |
-| Storage Array | PowerMax 2000/8000 <br> PowerMax 2500/8500 <br> 5978.479.479, 5978.711.711, 6079.xxx.xxx, Unisphere 10.0 | 1.0.x, 2.0.x, 2.1.x | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3 |
+|               |                                                 PowerMax                                                 |     PowerStore      |             PowerScale             | PowerFlex |
+| ------------- | :------------------------------------------------------------------------------------------------------: | :-----------------: | :--------------------------------: | :-------: |
+| Storage Array | PowerMax 2000/8000 <br> PowerMax 2500/8500 <br> 5978.479.479, 5978.711.711, 6079.xxx.xxx, Unisphere 10.0 | 1.0.x, 2.0.x, 2.1.x | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3 | 3.6       |
```

Comment:
do we support replication on other model (3.4, 3.5) of power flex arrays?

Reply:
Targetted and verified for PowerFlex 3.6 and 4.0.